scripts/es-integration-test.sh

```diff
@@ -20,75 +20,49 @@ check_arg() {
 
 setup_es() {
   local tag=$1
-  local image=docker.elastic.co/elasticsearch/elasticsearch
-  local params=(
-    --detach
-    --publish 9200:9200
-    --env "http.host=0.0.0.0"
-    --env "transport.host=127.0.0.1"
-    --env "xpack.security.enabled=false"
-  )
   local major_version=${tag%%.*}
-  if (( major_version < 8 )); then
-    params+=(--env "xpack.monitoring.enabled=false")
-  else
-    params+=(--env "xpack.monitoring.collection.enabled=false")
-  fi
-  if (( major_version > 7 )); then
-    params+=(
-      --env "action.destructive_requires_name=false"
-    )
-  fi
-
-  local cid
-  cid=$(docker run "${params[@]}" "${image}:${tag}")
-  echo "cid=${cid}" >> "$GITHUB_OUTPUT"
```

Comment:
you should keep this otherwise the workflow fails on the next step when it calls `docker logs` (which you need to change to `docker compose logs`)

Reply:
i added it  can u run the test to see if it works